package.json

```diff
@@ -84,7 +86,7 @@
     "watch:css": "yarn build:css && node-sass-chokidar --include-path ./src --include-path ./node_modules src/styles/index.scss -o src/styles/.css --watch --recursive",
     "build:js": "react-scripts build",
     "start": "node server.js",
-    "start:dev": "FUSE_URL=http://localhost:3006 LAUNCHER_URL=http://localhost:3006 ENMASSE_URL=http://localhost:3006 CHE_URL=http://localhost:3006 APICURIO_URL=http://localhost:3006 run-p -l watch:css start:local start:server",
+    "start:dev": "FUSE_URL=http://localhost:3006 LAUNCHER_URL=http://localhost:3006 ENMASSE_URL=http://localhost:3006 CHE_URL=http://localhost:3006 APICURITO_URL=http://localhost:3006 run-p -l watch:css start:local start:server",
```

Comment:
why is this necessary for this specific PR? last time the name was changed it broke OS4. make sure to test on OS4 to verify it doesn't break anything. yes it should never be hit because it has the dev flag but... test. also, in openshift-template.yml it is still APICURIO_URL not APICURITO_URL, so unsure why this is being changed.

Reply:
This should be removed.